src/types/__tests__/effector/sample/sample.test.ts

```diff
@@ -158,7 +300,14 @@ test('generic edge cases', () => {
   }
   expect(typecheck).toMatchInlineSnapshot(`
     "
-    no errors
+    Type 'Store<[any]> | Store<[any, ...any[]]> | Event<any>' is not assignable to type 'Store<A>'.
```

Comment:
regression

Reply:
Yep, I'm aware of this. The error appeared again after fixing the types for clock without source. I've tried to fix it, but so far without success.

Reply:
That's a task for further releases, not for `21.8.0`

Reply:
done